lib/crowbar/client/app/upgrade.rb

```diff
@@ -219,12 +219,12 @@ def backup(component)
           catch_errors(e)
         end
 
-        desc "repocheck ADDON",
-          "Check for existing repositories for an addon (ha|storage)"
+        desc "repocheck COMPONENT",
+          "Check for existing repositories for a component (crowbar|ha|storage)"
```

Comment:
> Not really :-(

then how would you name it?

> Could you explain in long_desc, what's crowbar value supposed to mean, so that potential user does not have to search the code?

good idea, I will add some more description